### ISSUE TITLE
`deleteCookie()` function does not return the deleted value

### ISSUE THREAD

**Issue by injoonH** [honojs/hono#2687]
### What version of Hono are you using?

4.2.6

### What runtime/platform is your app running on?

Bun

### What steps can reproduce the bug?

```ts
import { Hono } from 'hono'
import { deleteCookie } from 'hono/cookie'

const app = new Hono()

app.post('/', (c) => {
  const user = deleteCookie(c, 'user')
        // ^? const user: void
  return c.text('Signed out', 200)
})
```

### What is the expected behavior?

According to the [documentation](https://hono.dev/helpers/cookie#deletecookie), `deleteCookie()` function should return the deleted value.

### What do you see instead?

Contrary to the documentation, the `deleteCookie()` function is defined with a return type of `void`.

```ts
export declare const deleteCookie: (c: Context, name: string, opt?: CookieOptions) => void;
```

**Comment by fzn0x** [comment]
Looks like it's available in `4.3.0` https://github.com/honojs/hono/pull/2512.

Can you upgrade your Hono version?

**Comment by injoonH** [comment]
> Looks like it's available in `4.3.0` #2512.

By updating `hono` to the latest version (4.3.7), the problem has been resolved. Thanks for the help, @fzn0x! 😄

**Comment by fzn0x** [comment]
> > Looks like it's available in `4.3.0` #2512.
> 
> By updating `hono` to the latest version (4.3.7), the problem has been resolved. Thanks for the help, @fzn0x! 😄

Your welcome, glad to help 👍
